pkg/kyverno/apply/command.go

```diff
@@ -170,9 +186,11 @@ func Command() *cobra.Command {
 				}
 			}
 
-			resources, err := getResources(policies, resourcePaths, dClient)
-			if err != nil {
-				return sanitizedError.NewWithError("failed to load resources", err)
+			if resourcePaths[0] != "-" {
```

Comment:
can we move this up and consolidate with the prior if / else that sets the `resourcePaths` on line 132. This way all logic to set the `resourcePaths` is in a single place. We can then also extract into a method `setResourcePaths` for readability.